pages/team.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: MDAnalysis team
+---
+
+MDAnalysis is a community-driven project that is made possible through the efforts of many members who contribute in numerous and diverse ways, ranging from direct package development, maintenance, documentation, communication, and managerial responsibilities. On this page we list identified project roles and those historically responsible for the tasks. In the future, we plan to also list the contributors currently responsible for overseeing and managing associated tasks. We note that the listed roles on this page can differ significantly in scope and required effort.
+
+## Roles
+
+In this section we list contributors who have led or contributed substantial effort to the roles listed below.
+At present, we have only listed leading figures from the past 2-3 years, i.e. from 2020 onwards.
```

Comment:
```suggestion
At present, the current teams represent those who have been leading figures during the past 2-3 years (i.e., from 2020 onwards).
```

Reply:
Again, suggesting wording to be consistent with posting the current leads. We may also want to consider adding a line about what our process will be moving forward. For instance, do we want to do an annual (perhaps alongside the annual core dev roll call) review/update of the table? This way there would be an indication of the minimum time commitment someone is making by taking on a role.

Reply:
> We may also want to consider adding a line about what our process will be moving forward. For instance, do we want to do an annual (perhaps alongside the annual core dev roll call) review/update of the table? 

If that reflects the process we'd be happy to, but this might be something that all the core devs need to agree on first (unless it was discussed at the last core dev meeting) -- it doesn't seem like a decision that @fiona-naughton and I can make by ourselves. Given that the next core devs meeting is a while away, maybe we can resolve that here -- @MDAnalysis/coredevs, does an annual review of roles sound good to you? Does signing up for a role here imply a minimum commitment of a year?

Reply:
I've changed this text to reflect that we now have a "Current" column as well, and to add Jenna's note on when the team organisation structure was started.

Reply:
Annual review of the table sounds like a good idea, and we might as well do all of these "annual reviews" at the same time.

I am not so sure that this automatically implies the minimum commitment; that seems to be a more nuanced question which could go either way of 1 year.

Reply:
I've added this wording into the text -- nailing down minimum time commitments and role reviews may need to happen further down the line, so resolving this conversation for now!